steps/emulator-wtf-android-test/1.1.0/step.yml

```diff
@@ -0,0 +1,131 @@
+title: Run instrumentation tests on emulator.wtf
+summary: |
+  Run your Android UI tests with [emulator.wtf](https://emulator.wtf).
+description: |
+  Run your Android UI tests with [emulator.wtf](https://emulator.wtf).
+
+  Emulator.wtf is an Android cloud emulator laser-focused on performance to deliver quick feedback to your PRs.
+
+  It's like Firebase Test Lab on steroids (2-4x speedup, depending on your specific setup)
+website: https://emulator.wtf
+source_code_url: https://github.com/emulator-wtf/run-tests-bitrise
+support_url: https://github.com/emulator-wtf/run-tests-bitrise/issues
+published_at: 2021-12-20T13:42:00.870822+02:00
+source:
+  git: https://github.com/emulator-wtf/run-tests-bitrise.git
+  commit: 220673f48ab37de94f67ccb998d4e643ec2410fd
+project_type_tags:
+- android
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    title: emulator.wtf API Token
+  token: null
```

Comment:
Please add a summary to this input. (All inputs should have a title and summary at least, while the description is optional).